cocos/ui/UIEditBox/iOS/CCUISingleLineTextField.mm

```diff
@@ -29,7 +29,9 @@ of this software and associated documentation files (the "Software"), to deal
 #import "ui/UIEditBox/iOS/CCUITextInput.h"
 
 #include "base/CCDirector.h"
+#include <algorithm>
 
+#define RIGHT_VIEW_PADDING 5.0f
```

Comment:
`RIGHT_VIEW_PADDING ` -> `CAPS_LOCK_PADDING` is better, since it's only change padding  when caps lock is on